### PR TITLE
Interface/material editor

### PR DESCRIPTION
**Pull request by kwakuTM**
Documentation for Interface/Material-editor

Priority: low

### CODE REVIEW

File: interface/material-editor.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Material Editor
+The **Material Editor** allows us to create a **Material** that gives a **Mesh** a specific visual appearance, when it comes into contact with **Light** in a **Scene**. The **Material** files have a *.incmat* as their filename extension.
```
Comment (polygami) [inline interface/material-editor.md:2]: Materials define the shading model and parameters used when rendering 3D geometry.

Also, here is some text that I wrote for the newsletter. Maybe there is some information which can be added to the material-related documentation pages:

With the introduction of PBR (physically-based rendering) materials in Incari, 3D models can now be shaded in a much more standardized way, using HDRI environment maps to create realistic reflections and lighting behavior. In addition to the standard PBR shading model, Incari also offers a glass and car paint shader, as well as Phong and shadeless models.

File: interface/material-editor.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+In order to open the **Material Editor**, *right-click* on the *Menu bar* in Incari and *click* on **Material Editor** from the drop-down list. This can be seen in the illustration below:
```
Comment (polygami) [inline interface/material-editor.md:4]: Can also double click the material in the Asset Manager

File: interface/material-editor.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+![](../.gitbook/assets/interface/material-editor/material-editor.PNG)
+
+The sections defined in the **Material Editor** are discussed below:
+
+`Used Materials` section, you will find **Materials** that have previously applied to a **Mesh** in a **Scene**. These **Materials** can be edited by *clicking* on them.
+
+`New Material` as its name suggests allows us to create new **Materials**. The illustration below depicts its behavior when *clicked*.
```
Comment (polygami) [inline interface/material-editor.md:12]: I think these are only shown when no Material is opened right?

**Review by polygami — COMMENTED** [review]
I will write a detailed breakdown of all the shading models and their attributes etc. For now, just leave that out and I will send it to you once I get time!

File: interface/material-editor.md (hunk 1 above)
Comment (kwakuTM) [reply interface/material-editor.md:2]: I was thinking the will be a separate documentation to discuss just **Material**s with all the above information.

**Pull request merged**